cachito/web/api_v1.py

```diff
@@ -406,6 +410,14 @@ def patch_request(request_id):
                 "Failed to delete the temporary files at %s", bundle_dir
             )
 
+    if delete_logs:
+        request_log_dir = flask.current_app.config["CACHITO_REQUEST_FILE_LOGS_DIR"]
+        path_to_file = os.path.join(request_log_dir, f"{request_id}.log")
+        try:
+            os.path.remove(path_to_file)
+        except:  # noqa E722
```

Comment:
This is generally a bad idea. We should catch specific exceptions here such as `OSError`.

Otherwise, we may be concealing important, unrelated errors, such as `AttributeError`   that is always being passed here due to the bogus call to `os.path.remove`.

After this is changed, make sure to add a unit test that checks the file was indeed present before, and removed after passing through this code try/except block.

Reply:
I can't find OSError in `kombu.exceptions`. I'd like to explore that a bit if I can find something similar in there The same is repeated in the two sections right above this as well. I will create a separate PR for fixing them and provide coverage for all delete methods as a tech debt as none exists right now and it'd be a good idea to do it for all of them.